internal/cmd/build-iso.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/kairos-io/AuroraBoot/deployer"
+	"github.com/kairos-io/AuroraBoot/pkg/schema"
+	"github.com/spectrocloud-labs/herd"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	KairosDefaultArtifactName = "kairos"
+)
+
+var BuildISOCmd = cli.Command{
+	Name:    "build-iso",
+	Aliases: []string{"b"},
+	Usage:   "Builds an ISO from a container image or github release",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "cloud-config",
+			Aliases: []string{"c"},
+			Usage:   "The cloud config to embed in the ISO",
+		},
+		&cli.StringFlag{
+			Name:    "name",
+			Aliases: []string{"n"},
+			Usage:   "Basename of the generated ISO file",
+		},
+		&cli.StringFlag{
+			Name:    "output",
+			Aliases: []string{"o"},
+			Usage:   "Output directory (defaults to current directory)",
+		},
+		&cli.BoolFlag{
+			Name:  "date",
+			Value: false,
+			Usage: "Adds a date suffix into the generated ISO file",
+		},
+		&cli.StringFlag{
+			Name:  "overlay-rootfs",
+			Usage: "Path of the overlayed rootfs data",
+		},
+		&cli.StringFlag{
+			Name:  "overlay-uefi",
+			Usage: "Path of the overlayed uefi data",
+		},
+		&cli.StringFlag{
+			Name:  "overlay-iso",
+			Usage: "Path of the overlayed iso data",
+		},
+		&cli.StringFlag{
+			Name:  "label",
+			Usage: "Label of the ISO volume",
+		},
+		&cli.BoolFlag{
+			Name:  "squash-no-compression",
```

Comment:
same, would drop this. This has to be the default

Reply:
done